chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -35,46 +32,58 @@ class CurrentUser(
     private val chatManager: ChatManager
 ) {
 
+    val rooms: List<Room> get() = chatManager.roomStore.toList()
+        .filter { it.memberUserIds.contains(id) }
+    val users: Future<Result<List<User>, Error>>
```

Comment:
Does the `UserService` store a local copy of users that the SDK knows about? Could that be returned here if so?

Reply:
What this does is reply on the `RoomService`  and `ChatManager` to resolve the rooms. The rooms are stored on `ChatManager` and the `Service` will fill the store if we don't have a requested room. Hence the `future` here. If all the rooms requested are in memory then this will be instant.